RSFormView/Classes/Views/Form/FormView.swift

```diff
@@ -90,20 +90,21 @@ extension FormView: UITableViewDelegate, UITableViewDataSource {
                  cellForRowAt indexPath: IndexPath) -> UITableViewCell {
     
     let rowFields = viewModel?.items[indexPath.row].formFields
-    
-    if let formItem = viewModel?.items[indexPath.row],
-      let _ = formItem.attributedText,
-        rowFields?.isEmpty ?? true,
-        let cell = tableView
-          .dequeueReusableCell(withIdentifier: FormTextCell.reuseIdentifier,
-                               for: indexPath) as? FormTextCell {
-        cell.update(withFormItem: formItem, formConfigurator: formConfigurator)
-        return cell
-      }
-    
-    return textFieldCell(forRowAt: indexPath,
-                         in: tableView,
-                         with: rowFields)
+
+    guard let formItem = viewModel?.items[indexPath.row] else {
+      return UITableViewCell()
```

Comment:
I think we should avoid to return `UITableViewCell()` since it will cause a crash.
Maybe the `reuseIdentifier(forRowAt..)` method can return the default identifier for nil `formItems`.
Then casting the cell to `FormTableViewCell` should be done aside of the `dequeue` and we will return always a registered cell.